### ISSUE TITLE
Widget in miniplayer refreshes which causes an error.

### ISSUE THREAD

**Issue by eiler21** [peterscodee/miniplayer#22]
I asked the same [question ](https://stackoverflow.com/questions/68462735/videoplayer-becomes-black-when-dismissing-a-miniplayer-in-flutter/68499521?noredirect=1#comment122811637_68499521)on stackoverflow and after that found a solution. The problem is that miniplayer rebuilds  its child widget when you start to dismiss it because of this line:
`(value == 0) return child!;`

This causes an error when for example using video_player. Because at the time you start dismissing the miniplayer the video is playing however then miniplayer rebuilds its child and the video_player gets black (or starts playing over from the start).

If you just delete the line I mentioned above nothing will change and the error will disappear.

**Comment by dxvid-pts** [comment]
Thank you for submitting. 
This should be fixed now but feel free to reopen if the issue still occurs.

**Comment by zeddyyz** [comment]
Thank you @eiler21 for providing a solution. The current version of the package still presents the issue for me using just_audio package. Next, I copied the code for the package and made it local to my project. I removed the line if (value == 0) return child!; as you suggested to stop the widget from refreshing between max and min states, however my just_audio player refreshes the audio source between max/min state. Was that the only fix that worked for you or any ideas would be greatly appreciated?
